views/index.pug

```diff
@@ -64,6 +64,7 @@ block prepend content
                 span.dropdown-icon.fas.fa-cloud
                 | Short Link
                 | &nbsp;
+                button.dropdown-icon.fa.fa-clipboard.ml-auto.pull-right.clip-icon(data-toggle="none")
```

Comment:
this looks super ugly too. I couldn't get it to appear on the right, and I'm not sure if would be any good anyway